lalrpop-test/src/lib.rs

```diff
@@ -723,3 +729,18 @@ fn error_issue_278() {
         }
     }
 }
+
+#[test]
+fn test_empty() {
+    empty::parse_Empty("").unwrap();
+}
+
+mod empty_tok_lib {
```

Comment:
however, you could just use the `util` tokenizer that the rest of the tests use (see e.g. `sub_table` and so forth)